python/lsst/ts/IntegrationTests/obssys2_disabled_enabled.py

```diff
@@ -0,0 +1,42 @@
+# This file is part of ts_IntegrationTests
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["ObsSys2DisabledEnabled"]
+
+from lsst.ts.IntegrationTests import BaseScript
+from .config_registry import registry
+
+
+class ObsSys2DisabledEnabled(BaseScript):
+    """Execute the given Main Telescope Standard or External
```

Comment:
Just remove Main Telescope from this comment as it doesn't really apply.

Reply:
Fixed